### PR TITLE
Remove JamesForks/badger dependency

### LINKED ISSUE #1537: [BLOCKER] Drop dependency on JamesForks/poser
We can't ship 2.2 while still needing that.

### PR DESCRIPTION
**Pull request by jbrooksuk**
Closes #1537

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -2082,16 +2082,16 @@
         },
         {
             "name": "mccool/laravel-auto-presenter",
-            "version": "4.2.1",
+            "version": "v4.2.1",
```
Comment (GrahamCampbell) [inline composer.lock]: @jbrooksuk Your composer cache or vendor folder still need deleting,.

Comment (jbrooksuk) [inline composer.lock]: I've done it numerous times.

Comment (jbrooksuk) [inline composer.lock]: Doing it again.

Comment (GrahamCampbell) [inline composer.lock]: DId you delete the lock file as well?

**Pull request merged**